.travis.yml

```diff
@@ -1,87 +1,81 @@
 language: php
 
-sudo: false
+sudo: true
+
+addons:
+  firefox: "47.0.1"
+  postgresql: "9.4"
+  apt:
+    packages:
+      - openjdk-8-jre-headless
+
+services:
+  - mysql
```

Comment:
Mysql is already present in travis images and just needs starting.
Postgres is also added to every container so that a choice of databases can be used by later configs.